### PR TITLE
[ko] Update outdated files in dev-1.22-ko.4 (M56-M70)

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Related issue: #30846

> * [ ]  M56.  `content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md`  | 1(+XS) 1(-)
> * [ ]  M57.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md`  | 1(+XS) 1(-)
> * [ ]  M58.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md`  | 16(+S) 0(-)
> * [ ]  M59.  `content/en/docs/tasks/administer-cluster/sysctl-cluster.md`  | 1(+XS) 0(-)
> * [ ]  M60.  `content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md`  | 6(+XS) 0(-)
> * [ ]  M61.  `content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md`  | 22(+S) 15(-)
> * [ ]  M62.  `content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md`  | 7(+XS) 10(-)
> * [ ]  M63.  `content/en/docs/tasks/job/parallel-processing-expansion.md`  | 5(+XS) 5(-)
> * [ ]  M64.  `content/en/docs/tasks/run-application/horizontal-pod-autoscale.md`  | 39(+M) 1(-)
> * [ ]  M65.  `content/en/docs/tasks/tls/managing-tls-in-a-cluster.md`  | 13(+S) 3(-)
> * [ ]  M66.  `content/en/docs/tasks/tools/included/optional-kubectl-configs-zsh.md`  | 1(+XS) 1(-)
> * [ ]  M67.  `content/en/docs/tutorials/clusters/apparmor.md`  | 2(+XS) 2(-)
> * [ ]  M68.  `content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html`  | 1(+XS) 1(-)
> * [ ]  M69.  `content/en/docs/tutorials/services/source-ip.md`  | 1(+XS) 1(-)
> * [ ]  M70.  `content/en/docs/tutorials/stateful-application/zookeeper.md`  | 1(+XS) 2(-)

/language ko

### CODE REVIEW

**Comment by jihoon-seo** [comment]
force-push: `content/ko/examples/pods/storage/pv-duplicate.yaml` 파일이 누락되어, 추가하고
commit, squash, force-push 하였습니다. 😊

File: content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -506,6 +504,45 @@ HPA를 암시적으로 비활성화할 수 있다. 대상의 의도한
...
+이 값이 설정되어 있으면, (예를 들어 `kubectl apply -f deployment.yaml` 명령으로) 
+오브젝트에 변경이 생길 때마다 쿠버네티스가 파드의 수를 
+`spec.replicas`에 기재된 값으로 조정할 것이다. 
+이는 바람직하지 않으며 HPA가 활성화된 경우에 문제가 될 수 있다.
+
+`spec.replicas` 값을 제거하면 1회성으로 파드 숫자가 줄어들 수 있는데, 
+이는 이 키의 기본값이 1이기 때문이다(레퍼런스: 
+[디플로이먼트 레플리카](/ko/docs/concepts/workloads/controllers/deployment#레플리카)). 
+값을 업데이트하면, 파드 1개를 제외하고 나머지 파드가 종료 절차에 들어간다. 
+이후의 배포 동작은 정상적으로 작동하며 롤링 업데이트 구성도 의도한 대로 동작한다.
```
Comment (jmyung) [inline content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md:521]:
```suggestion
이후의 디플로이먼트 애플리케이션은 정상적으로 작동하며 롤링 업데이트 구성도 의도한 대로 동작한다. 
```

**Review by jmyung — COMMENTED** [review]
@jihoon-seo  수고 많으셨습니다.

몇 가지 의견을 드립니다.

File: content/ko/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 2]
```diff
@@ -6,29 +6,36 @@ weight: 100
...
 ```shell
 docker login
 ```
 
-프롬프트가 나타나면, 도커 사용자 이름(username)과 비밀번호(password)를 입력하자.
+프롬프트가 나타나면, 도커 ID를 입력한 다음, 사용하려는 크리덴셜(액세스 토큰, 
+또는 도커 ID의 비밀번호)을 입력한다.
 
-로그인 프로세스는 권한 토큰 정보를 가지고 있는 `config.json` 파일을 생성하거나 업데이트 한다.
+로그인 프로세스를 수행하면 권한 토큰 정보를 가지고 있는 `config.json` 파일이 생성되거나 업데이트된다. [쿠버네티스가 이 파일을 어떻게 해석하는지](/ko/docs/concepts/containers/images#config-json) 참고한다.
```
[the remark this answers is not in the document]
Comment (jihoon-seo) [reply content/ko/docs/tasks/configure-pod-container/pull-image-private-registry.md:38]: #30848 에서 추가될 예정입니다 😊
( https://github.com/kubernetes/website/pull/30848/files#diff-556165fe1b2be43d593a23fd258d1c2bc35fcf83bdcd75b497d3770b632f804eR268 )

**Review by jihoon-seo — COMMENTED** [review]
@jmyung 리뷰 감사합니다 😊
제안해 주신 내용들을 반영하여 새 커밋을 찍었고,
2차 변경 사항만 별도로 확인하시기 편하도록 squash는 아직 하지 않았습니다.
적절한 시기에 squash 하겠습니다. 😊

**Comment by jmyung** [comment]
/lgtm 

수고하셨습니다! 😄

**Comment by jihoon-seo** [comment]
squash + force-push 하였습니다. 😊

**Pull request merged**